server/tests/maria.test.js

```diff
@@ -0,0 +1,78 @@
+const maria = require('mariadb');
+const axios = require('axios');
+
+let mdb, holder;
+
+describe('should be able to connect and communicate with maria', () => {
+  beforeAll(async () => {
+    mdb = await maria.createConnection({
+      host: 'localhost',
+      user: 'root',
+      password: 'mar1a',
+      database: 'hrr'
+    });
+  })
+  test('this should be invoke if before all is successful', () => {
+    expect('maria').toBe('maria');
+  });
+  test('should be able to get a playlist base on id', () => {
+    mdb
+      .query('SELECT _id FROM current_playlist WHERE _id = 0')
+      .then((results) => {
+        console.log('results', results);
+        expects(results).toBe(results);
+      });
+  });
+  test('should be able to make request on api and get the playlist', () => {
+    axios
+      .get('http://localhost:3002/api/playlist/1')
+      .then((results) => {
+        expects(results).toBe(results);
+      });
+  });
+  test('should be able to delete and insert song on the playlist', () => {
+    axios
+      .get('http://localhost:3002/api/playlist/4')
+      .then(async (results) => {
+        let parsed_results = JSON.parse.results;
```

Comment:
JSON.parse(results)?

Reply:
thanks eliana.. no shortcuts for me then.. lol..